src/library/autodj/dlgautodj.cpp

```diff
@@ -270,29 +275,30 @@ void DlgAutoDJ::fadeNowButton(bool) {
 }
 
 void DlgAutoDJ::toggleAutoDJButton(bool enable) {
-    AutoDJProcessor::AutoDJError error = m_pAutoDJProcessor->toggleAutoDJ(enable);
+    m_pAutoDJProcessor->toggleAutoDJ(enable);
+}
+
+// TODO If there's a way to migrate the translations move this
+// to AutoDJProcessor in order to keep this class minimal
+void DlgAutoDJ::autoDJError(AutoDJProcessor::AutoDJError error) {
     switch (error) {
     case AutoDJProcessor::ADJ_NOT_TWO_DECKS:
         QMessageBox::warning(nullptr,
                 tr("Auto DJ"),
                 tr("Auto DJ requires two decks assigned to opposite sides of the crossfader."),
                 QMessageBox::Ok);
-        // Make sure the button becomes unpushed.
-        pushButtonAutoDJ->setChecked(false);
```

Comment:
here and below: obsolete, since `AutoDJProcessor::autoDJStateChanged` signal >> `DlgAutoDJ::autoDJStateChanged` updates the GUI